libs/category/state/src/reducers/category/category.reducer.spec.ts

```diff
@@ -51,6 +52,7 @@ describe('Category | Category Reducer', () => {
       sort_options: [],
       total_products: null,
       product_ids: [],
+      daffState: DaffState.Stable,
     },
     categoryLoading: false,
     productsLoading: false,
```

Comment:
This PR is essentially a replacement (and better version) of lines 57 and 58. All of the selector and facade fields related to them should be deprecated.

Reply:
done